openapi.yml

```diff
@@ -3442,14 +3442,12 @@ paths:
       - in: query
         name: workflow_state
         schema:
-          type: string
-          enum:
-          - DONE
-          - NEW
-          - PRE_SECONDARY_ASSESSMENT
-          - REJECTED
-          - SECONDARY_ASSESSMENT
-          - TRIAGE
```

Comment:
I think this is not entirely correct as we lost information about possible values of the enum, this could be easily solved if you use a different filter for the workflow_state:
```python
from django_filters.rest_framework import ChoiceFilter, BaseInFilter

class ChoiceInFilter(BaseInFilter, ChoiceFilter):
    """
    Filter for choice csv
    """

    pass
    
workflow_state = ChoiceInFilter(field_name="workflow_state")
```

Reply:
Actually this might not work correctly, there also exist `MultipleChoiceFilter`, not sure how that works, just pointing out that enum values should be exposed in the schema